packages/wonder-blocks-timing/src/hooks/__tests__/use-scheduled-timeout.test.js

```diff
@@ -224,7 +224,7 @@ describe("useScheduledTimeout", () => {
 
             // Assert
             expect(action).not.toHaveBeenCalled();
-            act(() => jest.advanceTimersByTime(500));
+            act((): void => jest.advanceTimersByTime(500));
```

Comment:
Flow couldn't decide between `() => void` and `() => Promise<void>`.  I'm not sure why given that `advanceTimersByTime()` doesn't return a promise. 🤷‍♂️